classes/checkout/CheckoutDeliveryStep.php

```diff
@@ -195,7 +195,16 @@ protected function isModuleComplete($requestParams)
     {
         $deliveryOptions = $this->getCheckoutSession()->getDeliveryOptions();
         $currentDeliveryOption = $deliveryOptions[$this->getCheckoutSession()->getSelectedDeliveryOption()];
-        if (!$currentDeliveryOption['is_module']) {
+
+        $isModule = false;
+
+        foreach ($currentDeliveryOption as $option) {
+            if ($option['is_module']) {
+                $isModule = true;
```

Comment:
```suggestion
                $isModule = true;
                break;
```
No need to continue if you found it :)